packages/editor/src/components/block-mover/icons.js

```diff
@@ -14,3 +14,11 @@ export const downArrow = (
 		<polygon points="9,13.5 14.7,7.9 13.2,6.5 9,10.7 4.8,6.5 3.3,7.9 " />
 	</AccessibleSVG>
 );
+
+export const dragHandle = (
+	<svg width="18" height="18" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 18 18" aria-hidden role="img" focusable="false">
```

Comment:
Why is this not implemented as `AccessibleSVG` like the others in this file? As best I can tell, it doesn't conflict as being one. And if it did, we should leave an explaining comment why it ought to be different.

Reply:
As Riad said. Didn't know that existed. Going to update, test, and commit.